web/src/beta/features/WorkspaceSetting/innerPages/Members/Members.tsx

```
@@ -0,0 +1,447 @@
+                    onClick={() => {
+                      setMemberSearchInput("");
+                      setDebouncedInput("");
+                      setMemberSearchResults(
+                        memberSearchResults.filter(
+                          (element) => element.id !== memberSearchResult.id
+                        )
+                      );
+                    }}
+                  />
+                </ItemContainer>
+              ))}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)